### PR TITLE
3.4.3 Add docs for EntityTrait::getDirty()

### PR DESCRIPTION
**Pull request by inoas**
Ref Feature Issue: https://github.com/cakephp/cakephp/issues/10292
Ref Feature PR: https://github.com/cakephp/cakephp/pull/10366

### CODE REVIEW

**Comment by inoas** [comment]
Ref Feature Issue: https://github.com/cakephp/cakephp/issues/10292
Ref Feature PR: https://github.com/cakephp/docs/pull/4796

File: en/orm/entities.rst [hunk 1]
```diff
@@ -239,6 +239,13 @@ by passing an extra option::
 
     $article = new Article(['title' => 'New Article'], ['markClean' => true]);
 
+.. versionadded:: 3.4.3
```
Comment (markstory) [inline en/orm/entities.rst]: Generally we put these annotations under the new feature.

**Comment by inoas** [comment]
Good?

**Comment by markstory** [comment]
Thanks!

**Pull request merged**